web_client/panels/DrawWidget.js

```diff
@@ -50,6 +53,14 @@ var DrawWidget = Panel.extend({
                 this._groups.get(this._style.id).save();
             }
         });
+        this.on('h:mouseover', (model) => {
+            this._highlighted[model.id] = true;
+            this.$(`.h-element[data-id="${model.id}"]`).addClass('h-highlight-element');
+        });
+        this.on('h:randomevent', (model) => {
```

Comment:
Why `h:randomevent` and not `h:mouseout`?  And, to that end, `h:mouseover` is fired on `mouseon` rather than `mouseover`.  Should its name reflect that?

Reply:
I honestly don't know where that event name came from.